### PR TITLE
implementmongodb

### PR DESCRIPTION
**Pull request by roabram**

### CODE REVIEW

File: src/app/components/CandleStickChart/CandleStickChart.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,6 @@
-import React from 'react';
 import CandleStickChart from './CandleStickChart';
 
 export default {
   title: 'Component/CandleStickChart',
   component: CandleStickChart,
 };
-
-export const Apple = (): JSX.Element => <CandleStickChart />;
```
Comment (lajanzen) [inline src/app/components/CandleStickChart/CandleStickChart.stories.tsx:9]: Maybe add a component here?

**Review by lajanzen — APPROVED** [review]
🚀

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

**Pull request merged**